tests/test_project.py

```diff
@@ -83,6 +83,40 @@ async def test_project_api_v1(client):
     assert response.code == 404
 
 
+async def test_delete_environment_project(client):
```

Comment:
This method uses the variable `result` and `response` for the response from an API endpoint. Let's keep it consistent and use one of both names for that. Same in the other test case.